universal-application-tool-0.0.1/app/modules/SecurityModule.java

```diff
@@ -101,175 +95,72 @@ protected void configure() {
     PlayCookieSessionStore sessionStore =
         new PlayCookieSessionStore(new ShiroAesDataEncrypter(aesKey));
     bind(SessionStore.class).toInstance(sessionStore);
-  }
 
-  @Provides
-  @Singleton
-  protected GuestClient guestClient(ProfileFactory profileFactory) {
-    return new GuestClient(profileFactory);
+    // Default to these options
+    String applicantAuthClient = "adfs";
```

Comment:
question (blocking): shouldn't the applicant auth client default to IDCS?

Reply:
Ah yes thank you for catching that!